enforcer.go

```diff
@@ -38,10 +39,11 @@ type Enforcer struct {
 	fm        model.FunctionMap
 	eft       effector.Effector
 
-	adapter    persist.Adapter
-	watcher    persist.Watcher
-	dispatcher persist.Dispatcher
-	rmMap      map[string]rbac.RoleManager
+	adapter     persist.Adapter
+	watcher     persist.Watcher
+	dispatcher  persist.Dispatcher
+	rmMap       map[string]rbac.RoleManager
+	expressions sync.Map
```

Comment:
We should declare ``expressions`` as a variable, not a member of the enforcer structure

Reply:
@tangyang9464 you mean global var? Why not an enforcer member?

Reply:
I made a mistake. It's ok.